src/test/java/de/plushnikov/intellij/plugin/processor/UtilityClassTest.java

```diff
@@ -0,0 +1,17 @@
+package de.plushnikov.intellij.plugin.processor;
+
+import com.intellij.openapi.application.ApplicationInfo;
+import com.intellij.openapi.util.BuildNumber;
+import de.plushnikov.intellij.plugin.AbstractLombokParsingTestCase;
+
+import java.io.IOException;
+
+public class UtilityClassTest extends AbstractLombokParsingTestCase {
+  protected boolean shouldCompareModifiers() {
+    return ApplicationInfo.getInstance().getBuild().compareTo(BuildNumber.fromString("146.1154")) >= 0;
+  }
+
+  public void testUtilityClassPlain() throws IOException {
+    doTest(true);
```

Comment:
You are working on windows, right :-)? 
Windows is caseinsensetive, but linux is casesensitive system. You have to call doTest(false) because your testdatafile is named "UtilityClassPlain.java".
Or better, plase your testdatafile into one subdirectory "utilityclass", and then change your testmethod name to "testUtilityClass$UtilityClassPlain".